src/vg.cpp

```diff
@@ -1842,6 +1846,15 @@ void VG::from_alleles(const map<long, set<vcflib::VariantAllele> >& altp,
 
             }
 
+            if (alt_node != NULL) {
+                auto nameIter = alleleNames.find(allele);
+                if (nameIter != alleleNames.end()) {
+                    for (auto& name : nameIter->second) {
+                        paths.append_mapping(name, alt_node->id());
```

Comment:
It looks like this is just putting mappings on the non-reference nodes involved in the alt. If the alt has any matches to the reference separating the alleles representing bubbles, this will produce an invalid path, right?